Makefile

```
@@ -1,6 +1,6 @@
 DATE		= $(shell date)
-PYTHON		= $(shell poetry run which python 2>/dev/null || which python)
-PKG_VERSION = $(shell poetry -s version)
+PYTHON		= $(shell uv run which python 2>/dev/null || which python)
+PKG_VERSION = $(shell uv run python get-version.py)
```

optional, since we have `yq` installed for some quipucords make targets

```suggestion
PKG_VERSION = $(shell yq '.project.version' pyproject.toml)
```

---

I thought about this too, but I didn't feel like declaring `yq` elsewhere like the readme and the github workers because using `yq` here would make it required for _anything_ that uses _any_ `make` commands, whereas with quipucords I think `yq` is only really needed for the Konflux pipeline. Maybe later if/when Konflux starts building qpc… 😉